renderers/legend_label_offset.map

```diff
@@ -0,0 +1,247 @@
+# RUN_PARMS: legend.png [MAPSERV] QUERY_STRING="map=[MAPFILE]&mode=legend&map.imagetype=png" > [RESULT_DEMIME]
```

Comment:
this needs to be renamed from ```legend.png``` to ```legend_label_offset.png``` in order to create a new test case (the current code would test your mapfile against the already exisiting legend.png.